tap_egencia/secret.py

```diff
@@ -0,0 +1,30 @@
+# import os
```

Comment:
Looks like we aren't using this to actually pull secrets, can we get rid of this file?

Reply:
will do